container/crio/client.go

```diff
@@ -47,13 +47,28 @@ type ContainerInfo struct {
 	IP          string            `json:"ip_address"`
 }
 
+// containerInfoDualStack corresponds to cri-o release which supports dual stack
+type containerInfoDualStack struct {
```

Comment:
This wrapper seems strange.  I'd prefer just adding to the ContainerInfo struct.  I believe it should still be able to unmarshal from older versions...

Reply:
@dashpole 
Comment has been addressed.